src/challenge/challenge.resolver.ts

```diff
@@ -0,0 +1,19 @@
+import { Resolver, Mutation, Args } from '@nestjs/graphql';
+import { ChallengeEntity as Challenge } from './entities/challenge.entity';
+import { ChallengeService } from './challenge.service';
+import { UseGuards } from '@nestjs/common';
+import { GqlJwtAuthGuard } from 'src/common/guards/gql-jwt-auth.guard';
+import { CreateChallengeDto } from './dto/create-challenge.dto';
+import { UpsertChallengeDto } from './dto/upsert-challenge.dto';
+
+@Resolver(() => Challenge)
+export class ChallengesResolver {
+  constructor(private readonly challengeService: ChallengeService) {}
+
+  @Mutation(() => Challenge)
```

Comment:
Challenges will be created when importing the GEdIL package, their specification is inside. CRUD mutations are not needed ;-)